libpod/container_api.go

```diff
@@ -667,13 +667,15 @@ func (c *Container) Batch(batchFunc func(*Container) error) error {
 
 	newCtr.batched = true
 
-	if err := batchFunc(newCtr); err != nil {
-		return err
-	}
+	err := batchFunc(newCtr)
+	//if err := batchFunc(newCtr); err != nil {
+	//	return err
+	//}
 
 	newCtr.batched = false
 
-	return c.save()
+	//return c.save()
+	return err
```

Comment:
Remove command and just do

return batchFunc(newCtr)

Reply:
I believe it needs to be this way to so .batched can be switched back unless you want me to carve out a function to just switch it back so i can put a deferred call in.